README.md

```diff
@@ -196,14 +205,6 @@ print _brs_.global
 ' }
 ```
 
-### `_brs_.runInScope`
-
-Runs a file (or set of files) **in the current global + module scope** with the provided arguments, returning either the value returned by those files' `main` function or `invalid` if an error occurs.
-
-```brightscript
-_brs_.runInScope("/path/to/myFile.brs", { foo: "bar" })
-```
-
```

Comment:
This kind of reordering is fine for now, but next time let's shoot for a separate commit to reorder.  Or even better, a separate PR!  It'll help separate "things I've changed because feature X needs it that way" from "things I changed because they're better this way" 😃

Reply:
🙈 yep, great call -- i might actually just do that now!